recipes/cryptominisat/build.sh

```diff
@@ -0,0 +1,18 @@
+#!/bin/bash
+set -eu -o pipefail
+
+cd "$SRC_DIR"
+
+mkdir -p build && cd build
+cmake \
+  "-DCMAKE_INSTALL_PREFIX=$PREFIX" \
+  -DSTATICCOMPILE=ON \
```

Comment:
Because of this, it looks like it's searching for a static pthreads version, but fails to find it

Reply:
Ok. Let me try.